mutters-core/src/main/java/com/rabidgremlin/mutters/core/MatcherScores.java

```diff
@@ -42,37 +47,32 @@ public MatcherScores(SortedMap<Double, SortedSet<String>> scores)
 
   /**
    * Add a score for an intent to the matcher scores.
-   * 
+   *
    * @param intent The name of the intent to add the score for.
    * @param score  The score for the intent.
    * @return returns this.
    */
   public MatcherScores addScore(String intent, Double score)
   {
+    // TODO what if there are duplicate scores for some intent?
```

Comment:
Yeah, this is probably the biggest problem with the change. What do we do with the dups ???